### DISCUSSION TITLE
Breaking changes made in 22000.282

### DISCUSSION THREAD

**Discussion by BraINstinct0** [valinet/ExplorerPatcher#133] [General]
Behaviors noted:
- [Symbols not downloading](https://github.com/valinet/ExplorerPatcher/discussions/123#discussion-3628364)
- Seems like [Task bar & Start menu edit not applied](https://github.com/valinet/ExplorerPatcher/issues/129)
- right-click context menu edit is applied correctly [^1]
- right-click on start menu icon (on taskbar) doesn't show the ExplorerPatcher setting option
- Explorer search tweak(to Win10 version) seems to partially work. Not sure, so other users might confirm it for me.

Some more info/reports could be added as comments?
---
The list of 'fixes' made on this 282 update is [here](https://blogs.windows.com/windows-insider/2021/10/15/releasing-windows-11-build-22000-282-to-beta-and-release-preview-channels/].

One thing that seems interesting is:
* We fixed an issue for a small number of users that prevents the Start menu from working and prevents you from seeing the updated taskbar design after upgrading to Windows 11 (original release).

---
[^1]:![right-click context menu edit is applied correctly](https://user-images.githubusercontent.com/29201475/137587802-a9e61030-fc76-4040-9af2-96a4caab88aa.png)

**Comment by BraINstinct0** [comment]
- Win+C as Clock flyout doesn't work anymore. Instead, it shows the Teams.

**Reply by valinet** [reply]
That's probably the cause. That function that I hooked was removed, symbols download process fails, and it cannot proceed from there.

**Reply by valinet** [reply]
Even worse. The symbol files seem not to be available yet. That's a bummer...
